backend/.env.dist

```
@@ -21,6 +21,10 @@ KLICKTIPP_PASSWORD=secret321
 KLICKTIPP_APIKEY_DE=SomeFakeKeyDE
 KLICKTIPP_APIKEY_EN=SomeFakeKeyEN
 
+# DltConnector
+DLT_CONNECTOR=true
+DLT_CONNECTOR_URL=http://localhost:6000
```

```suggestion
DLT_CONNECTOR_URL=http://localhost:6010
```
Port 6000 is used under linux from x-windows